types/index.d.ts

```diff
@@ -399,6 +400,11 @@ export namespace Server {
     interface Options {
         allowHttp: boolean;
         headers?: Map<string, string>;
+        retry?: {
+            retries?: number,
+            retryDelay?: Function,
```

Comment:
add function declaration, i.e. `(retryCount: number, error: axios.AxiosError) => number`,
you can see `axios-retry` exact declaration in their `index.d.ts`.